simulator/src/test/java/com/hedera/block/simulator/config/TestConfigBuilder.java

```diff
@@ -0,0 +1,242 @@
+     *
+     * @param registerAllTypes if true all config data records on classpath will automatically be
+     *     registered
+     */
+    public TestConfigBuilder(final boolean registerAllTypes) {
+        if (registerAllTypes) {
+            this.builder = ConfigurationBuilder.create().autoDiscoverExtensions();
+        } else {
+            this.builder = ConfigurationBuilder.create();
+        }
+    }
+
+    /**
+     * Sets the value for the config.
+     *
+     * @param propertyName name of the property
+     * @param value the value
+     * @return the {@link TestConfigBuilder} instance (for fluent API)
+     */
+    @NonNull
+    public TestConfigBuilder withValue(
+            @NonNull final String propertyName, @Nullable final String value) {
+        return withSource(new SimpleConfigSource(propertyName, value));
+    }
+
+    /**
+     * Sets the value for the config.
+     *
+     * @param propertyName name of the property
+     * @param value the value
+     * @return the {@link TestConfigBuilder} instance (for fluent API)
+     */
+    @NonNull
+    public TestConfigBuilder withValue(@NonNull final String propertyName, final int value) {
+        return withSource(new SimpleConfigSource(propertyName, value));
+    }
+
+    /**
+     * Sets the value for the config.
+     *
+     * @param propertyName name of the property
+     * @param value the value
+     * @return the {@link TestConfigBuilder} instance (for fluent API)
+     */
+    @NonNull
+    public TestConfigBuilder withValue(@NonNull final String propertyName, final double value) {
+        return withSource(new SimpleConfigSource(propertyName, value));
+    }
+
+    /**
+     * Sets the value for the config.
+     *
+     * @param propertyName name of the property
+     * @param value the value
+     * @return the {@link TestConfigBuilder} instance (for fluent API)
+     */
+    @NonNull
+    public TestConfigBuilder withValue(@NonNull final String propertyName, final long value) {
+        return withSource(new SimpleConfigSource(propertyName, value));
+    }
+
+    /**
+     * Sets the value for the config.
+     *
+     * @param propertyName name of the property
+     * @param value the value
+     * @return the {@link TestConfigBuilder} instance (for fluent API)
+     */
+    @NonNull
+    public TestConfigBuilder withValue(@NonNull final String propertyName, final boolean value) {
+        return withSource(new SimpleConfigSource(propertyName, value));
+    }
+
+    /**
+     * Sets the value for the config.
+     *
+     * @param propertyName name of the property
+     * @param value the value
+     * @return the {@link TestConfigBuilder} instance (for fluent API)
+     */
+    @NonNull
+    public TestConfigBuilder withValue(
+            @NonNull final String propertyName, @NonNull final Object value) {
+        Objects.requireNonNull(value, "value must not be null");
+        return withSource(new SimpleConfigSource(propertyName, value.toString()));
+    }
```

Comment:
Several of these methods look like they're unused. Should they be part of this PR?

Reply:
yeah, this is another Copy/Pasta for testing purposes, this is only for testing capabilities, the original source is `services` they have it as part of a test-fixtures class, but they don't publish that, so had to do the copy/pasta onto server project, and now into simulator aswell, I would preffer to keep it the same, so we know is the same functionality and ability to perform future config tests across the 3 projects.